src/znc.cpp

```diff
@@ -628,6 +629,7 @@ bool CZNC::WriteNewConfig(const CString& sConfigFile) {
     VCString vsLines;
 
     vsLines.push_back(MakeConfigHeader());
+    vsLines.push_back("PidFile = /run/znc/znc.pid");
```

Comment:
While I do not really understand your concerns regarding picking up compiled znc and move to another system (these people shouldn't set the option per default so they don't end up with a default PidFile option) let me tell you why I initially added a default PidFile option:

If you are using a runscript/unit file you need a known resource you can check for status. I.e. if znc is running there must be a pidfile and this pid must be alive (just checking for znc processes isn't enough when you want to support multiple instances).

When user are installing znc, the configuration tool don't set such an option per default. So this is a problem. Telling the user "Now that you called the config tool you still need to add something manually" is not a good user experience, is it? That's why I added the patch.

We could only set PidFile option per default when `bSystemWideConfig` is true. However you would still disagree because this could be a problem if people decide to copy the compiled binary to another system where they want to use a different path.

Another solution would be to add a pidfile command line option. This way we could set the pidfile via runscript to a known value and wouldn't care if the config option is set and matches (cli parameter overrides).

What do you think?